src/sentry/demo/data_population.py

```diff
@@ -67,6 +67,7 @@
 base_paths_by_file_type = {"js": ["components/", "views/"], "py": ["flask/", "routes/"]}
 
 rate_by_release_num = [1.0, 0.99, 0.9]
+threshold_by_release_num = [0.004, 0.0015, 0.01]
```

Comment:
@roggenkemper `rate_by_release_num` is used for a similar purpose but for non-aggregate sessions. We should update variable names to reflect this. Also, we should either do error rates or success rates but not both.

Reply:
I just combined the two, they do practically the same thing.

Reply:
@roggenkemper the problem with combining them is that with single sessions, if the error rate is too low we might not get _any_ instances of that error. So we should let the individual session errors have a higher error rate since we won't have as many sessions.

Reply:
we can make it a math equation though and have one constant array

Reply:
I think keeping it as 2 separate and different valued arrays might be easier than a math equation and a constant array.